### DISCUSSION TITLE
Add topics-tags to help with project discoverability

### DISCUSSION THREAD

**Discussion by zigzag312** [exograd/eventline#8] [General]
I have been researching what open source job scheduler projects are available and Eventline didn't show up in any results of my Google and GitHub queries. Somehow I did manage to find your submission from 2022 on HN. Maybe adding more [GitHub topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics) would help with project discoverability? 

I does look like a nice, lightweight solution that even comes with integrated UI and monitoring.

**Comment by galdor** [comment]
I do not disagree :) There are already a couple tags; which ones would you suggest I add?

**Reply by zigzag312** [reply]
Some tags I have seen used in other projects: 

`scheduled-jobs` `background-jobs` `jobs` `cron` `scheduler` `scheduling` `scheduled-tasks` `trigger` `task` `task-scheduler` `task-manager` `job-management` `runner` `recurring` `interval` `job-processor`
